pkg/destroy/aws/aws.go

```diff
@@ -33,7 +33,9 @@ import (
 )
 
 var (
-	exists = struct{}{}
+	exists      = struct{}{}
+	currentTime = time.Now()
```

Comment:
This variable is never set anywhere but here.

Reply:
I am also updating the variable at L203?

Reply:
No, you are creating a new `currentTime` variable at that line.

Reply:
Got it now. Thanks